scripts/install_torchbenchmark.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+if [ "$1" = "-h" -o "$1" = "--help" ]; then
+  echo "Usage: prog BRANCH_NAME REPO"
+  exit 1
+fi
+
+if [ "$#" -ne 2 ]; then
+    echo echo "Usage: prog BRANCH_NAME REPO"
+    exit 1
+fi
+
+. ${CONDA_EXE%bin/conda}/etc/profile.d/conda.sh
+ORIG_BRANCH_NAME=$1
+REPO=$2
+BRANCH_NAME=$(python -c 'import sys; print (sys.argv[1].replace("/","_"))' $ORIG_BRANCH_NAME)
+echo "Your branch will be built into $BRANCH_NAME and torchbenchmark will be installed in ${BRANCH_NAME}/benchmark"
+conda create -y -n $BRANCH_NAME python=3.7 2>&1 > /dev/null
+test $? -eq 0 || { echo "failed to create env ${BRANCH_NAME}!"; exit; }
+conda activate $BRANCH_NAME
+echo "Installing PyTorch dependencies"
+conda install -y numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses 2>&1 > /dev/null
```

Comment:
Maybe use `conda install --only-deps pytorch` to install only dependencies?

Reply:
@xuzhao9  let me try seeing if that works. I think for building there are more dependencies than listed in a conda package

Reply:
@xuzhao9 yup, it breaks :-( 
```
Building PyTorch
Traceback (most recent call last):
  File "setup.py", line 368, in check_pydep
    importlib.import_module(importname)
  File "/home/villedepommes/miniconda3/envs/krovatkin_shapes_grad/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'yaml'
```

Reply:
Ahh right, this is because `yaml` is a build-time dependency, not runtime dependency. In this case, we have to install the build time deps manually.